src/scripts/main.js

```diff
@@ -1,7 +1,45 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
+  notification.className = `notification ${type}`;
+  notification.style.position = 'relative';
```

Comment:
The style position is set to 'relative' for the notification, which might not be necessary since the container is already positioned as 'fixed'. Consider removing this line unless there is a specific reason for it.